example/android/javalib/1-hello-world/build.mill

```diff
@@ -52,14 +105,58 @@ object app extends AndroidAppModule {
 //
 // ----
 // .
-// ├── app
-// │   ├── AndroidManifest.xml
-// │   ├── releaseKey.jks
-// │   ├── resources
-// │   │   └── values
-// │   │       ├── colors.xml
-// │   │       └── strings.xml
-// │   └── src/main/java/com/helloworld/app/MainActivity.java
-// └── build.mill
+//├── app
+//│     └── src
+//│         ├── androidTest/java/com/helloworld/app/ExampleInstrumentedTest.java
+//│         ├── main
+//│         │     ├── AndroidManifest.xml
+//│         │     ├── java/com/helloworld/app/MainActivity.java
+//│         │     └── res
+//│         │         └── values
+//│         │             ├── colors.xml
+//│         │             └── strings.xml
+//│         └── test/java/com/helloworld/app/ExampleUnitTest.java
+//└── build.mill
 // ----
 //
+
+/** Usage
+
+> ./mill show app.test
+...compiling 1 Java source...
+
+> cat out/app/test/test.dest/out.json
+["",[{"fullyQualifiedName":"com.helloworld.ExampleUnitTest.textSize_isCorrect","selector":"com.helloworld.ExampleUnitTest.textSize_isCorrect","duration":...,"status":"Success"}]]
+
+*/
+
+// This command runs unit tests on your local environment.
+
+/** Usage
+
+> ./mill app.createAndroidVirtualDevice
+
+> ./mill show app.startAndroidEmulator
+
+> ./mill show app.waitForDevice
+...emulator-5554...
+
+> ./mill show app.it | grep '"OK (1 test)"'
+..."OK (1 test)",
+
+> cat out/app/it/test.json | grep '"OK (1 test)"'
+..."OK (1 test)"...
+
+> ./mill show app.stopAndroidEmulator
+
+> ./mill show app.deleteAndroidVirtualDevice
+
+*/
+
+// The android tests (existing typically in androidTest directory, aka integration tests)
```

Comment:
minor: it is better to call them instrumented tests to follow the official Android lingvo https://developer.android.com/training/testing/instrumented-tests, integration test for the Android module can be a pure JVM test as well without the need of the Android VM.

Moreover, the text in `ExampleInstrumentedTest` in your PR is using a word `instrumented`.

Reply:
thanks